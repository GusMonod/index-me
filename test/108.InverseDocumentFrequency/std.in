Hello welcome to this test
welcome to this test
to this test
this test
test
