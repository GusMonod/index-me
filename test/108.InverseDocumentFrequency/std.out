Hello: idf=0.70, in docs: 0(tf = 1)
welcome: idf=0.40, in docs: 0(tf = 1), 1(tf = 1)
to: idf=0.22, in docs: 0(tf = 1), 1(tf = 1), 2(tf = 1)
this: idf=0.10, in docs: 0(tf = 1), 1(tf = 1), 2(tf = 1), 3(tf = 1)
test: idf=0.00, in docs: 0(tf = 1), 1(tf = 1), 2(tf = 1), 3(tf = 1), 4(tf = 1)
